tendrl/gluster_integration/message/callback.py

```diff
@@ -9,14 +9,347 @@
+        message = "Minimum number of bricks back online " \
+                  "in EC subvolume: {}".format(
+                      event['message']['subvol']
+                  )
+        native_event = NS.gluster.objects.NativeEvents(
+            context,
+            message=message,
+            severity="recovery",
+            current_value="ec_min_bricks_up"
+        )
+        native_event.save()
+
+    def afr_quorum_met(self, event):
+        context = "afr_quorum_state" + event['message']['subvol']
+        message = "Afr quorum is met for subvolume: {}".format(
+            event['message']['subvol']
+        )
+        native_event = NS.gluster.objects.NativeEvents(
+            context,
+            message=message,
+            severity="recovery",
+            current_value="afr_quorum_met"
+        )
+        native_event.save()
+
+    def afr_quorum_fail(self, event):
+        context = "afr_quorum_state" + event['message']['subvol']
+        message = "Afr quorum has failed for subvolume: {}".format(
+            event['message']['subvol']
+        )
+        native_event = NS.gluster.objects.NativeEvents(
+            context,
+            message=message,
+            severity="warning",
+            current_value="afr_quorum_failed"
+        )
+        native_event.save()
+
+    def afr_subvol_up(self, event):
+        context = "afr_subvol_state" + event['message']['subvol']
+        message = "Afr subvolume: {} is back up".format(
+            event['message']['subvol']
+        )
+        native_event = NS.gluster.objects.NativeEvents(
+            context,
+            message=message,
+            severity="recovery",
+            current_value="afr_subvol_up"
+        )
+        native_event.save()
+
+    def afr_subvols_down(self, event):
+        context = "afr_subvol_state" + event['message']['subvol']
+        message = "Afr subvolume: {} is down".format(
+            event['message']['subvol']
+        )
+        native_event = NS.gluster.objects.NativeEvents(
+            context,
+            message=message,
+            severity="warning",
+            current_value="afr_subvol_down"
+        )
+        native_event.save()
+
+    def unknown_peer(self, event):
+        context = "unknown_peer" + event['message']['peer']
+        message = "Peer {} has moved to unknown state".format(
+            event['message']['peer']
+        )
+        native_event = NS.gluster.objects.NativeEvents(
+            context,
+            message=message,
+            severity="warning",
+            current_value="unknown_peer",
+            alert_notify=True
+        )
+        native_event.save()
+
+    def brickpath_resolve_failed(self, event):
+        context = "brickpath_resolve_failed" + event['message'][
+            "peer"] + event['message']["volume"] + event['message']["brick"]
+        message = "Brick path resolution failed for brick: {0} . Volume: {1}" \
+                  ".Peer: {}".format(
```

Comment:
Use numbered format strings in all the places.

Reply:
ack